pkg/eosclient/eosgrpc/eosgrpc.go

```diff
@@ -35,9 +34,9 @@ import (
 	"github.com/cs3org/reva/pkg/appctx"
 	"github.com/cs3org/reva/pkg/eosclient"
 	erpc "github.com/cs3org/reva/pkg/eosclient/eosgrpc/eos_grpc"
+	ehttp "github.com/cs3org/reva/pkg/eosclient/eosgrpc/eos_http"
```

Comment:
The proto files need to be added

Reply:
What do you mean here? Which proto file?